CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Changed
 
 - `ext.SpanTypeMessageConsumer` and `ext.SpanTypeMessageProducer` now evaluate to `consumer` and `producer` respectively instead of `queue`. ([#119](https://github.com/signalfx/signalfx-go-tracing/pull/119))
+- Sarama instrumentation requires Kafka version 0.11.0 or newer to work correctly. ([#120](https://github.com/signalfx/signalfx-go-tracing/pull/120))
+
+### Fixed
+
+- Sarama instrumentation correctly identifies and finishes spans. ([#120](https://github.com/signalfx/signalfx-go-tracing/pull/120))
```

Comment:
maybe we should remove `and finishes spans` 😉

Reply:
I don't think "removing spans" makes any sense to users so not sure if it belong in the changelog. That's more of an implementation detail and belongs to git commits IMO.

Reply:
I just mean to change like that: 

```suggestion
- Sarama instrumentation correctly identifies spans. ([#120](https://github.com/signalfx/signalfx-go-tracing/pull/120))
```